src/wwwroot/partials/settings/connection-settings.html

```diff
@@ -37,7 +37,7 @@
       <label>{{'connection-settings_api_key_title' | translate}}</label>
       <span class="api-key--container">
         <p>{{vm.apiKey}}</p>
-        <button class="button button--extra-small" ng-click="vm.showCopy()">{{'connection-settings_copy_api_key' | translate}}</button>
+        <copy-to-clipboard copy-target=".api-key--container p" copy-tooltip-message-key="connection-settings_copy_success" button-text-key="connection-settings_copy_api_key"></copy-to-clipboard>        
```

Comment:
@martinkeimel is `copy-target ` a standard angular attribute? 

In that case, @pbarrios, maybe it is useful to generate the _id_ in the inputs generated by the select plugin.